src/__tests__/App.test.js

```diff
@@ -0,0 +1,46 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import App from "../components/App";
+import mockApiResponse from "./mockApiResponse.json";
+import nock from "nock";
+
+describe("App", () => {
+  it("matches snapshot", () => {
+    const { asFragment } = render(<App />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders the NASA logo", () => {
+    render(<App />);
+
+    expect(screen.getByAltText("nasa logo")).toBeInTheDocument();
+  });
+
+  it("renders Search component", () => {
+    render(<App />);
+
+    expect(screen.getByRole("form")).toBeInTheDocument();
+  });
+
+  it("displays SearchResults component on successful search", async () => {
```

Comment:
This is actually an integration test rather than a unit test, and it is still actually calling the API (you can verify this by disconnecting your wifi and running the test, the error given will confirm it).
I would say you are already checking for this logic in `SearchResults.test.js` (the valid props passed in being providing the data the component needs) and the calling of the function in the `Search.test.js`